superset-frontend/src/explore/constants.js

```diff
@@ -53,10 +55,10 @@ export const HAVING_OPERATORS = [
   OPERATORS['>='],
   OPERATORS['<='],
 ];
-export const MULTI_OPERATORS = [OPERATORS.in, OPERATORS['not in']];
+export const MULTI_OPERATORS = new Set([OPERATORS.in, OPERATORS['not in']]);
```

Comment:
not changed here, but I believe @villebro standardized the operators to be `IN` and `NOT IN`. Perhaps that change should be made?

Reply:
Maybe in the separate PR if that's really necessary. I don't want to break things.